src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -851,6 +851,18 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
                 }
             }
 
+            // apply http configuration configuration
+            configSection = (JObject)config["http"];
+            string routePrefix = ScriptConstants.DefaultHttpRoutePrefix;
+            if (configSection != null)
+            {
+                if (configSection.TryGetValue("routePrefix", out value))
+                {
+                    routePrefix = (string)value;
+                }
```

Comment:
Might be a good idea to add a route that catches all requests to unregistered routes under `admin` and responds appropriately (404?)